util/envDynawo.sh

```diff
@@ -319,41 +320,41 @@ set_environment() {
 
   # Third parties
   export_var_env_force DYNAWO_THIRD_PARTY_SRC_DIR=$DYNAWO_SRC_DIR/3rdParty
-  export_var_env DYNAWO_THIRD_PARTY_BUILD_DIR=$DYNAWO_HOME/build/3rdParty/$DYNAWO_COMPILER_NAME$DYNAWO_COMPILER_VERSION/$(echo $DYNAWO_LIBRARY_TYPE | tr "[A-Z]" "[a-z]")
-  export_var_env DYNAWO_THIRD_PARTY_INSTALL_DIR=$DYNAWO_HOME/install/3rdParty/$DYNAWO_COMPILER_NAME$DYNAWO_COMPILER_VERSION/$(echo $DYNAWO_LIBRARY_TYPE | tr "[A-Z]" "[a-z]")
-  export_var_env_force DYNAWO_THIRD_PARTY_BUILD_DIR_VERSION=$DYNAWO_THIRD_PARTY_BUILD_DIR/$DYNAWO_BUILD_TYPE_THIRD_PARTY$SUFFIX_CX11
-  export_var_env_force DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION=$DYNAWO_THIRD_PARTY_INSTALL_DIR/$DYNAWO_BUILD_TYPE_THIRD_PARTY$SUFFIX_CX11
-
-  if [ -d "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost" ]; then
-    if [ ! -z "$(ls -A $DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost)" ]; then
-      export_var_env DYNAWO_BOOST_HOME=$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost
+  export_var_env_force DYNAWO_THIRD_PARTY_BUILD_DIR_VERSION=$DYNAWO_HOME/build/3rdParty/$DYNAWO_COMPILER_NAME$DYNAWO_COMPILER_VERSION/$(echo $DYNAWO_LIBRARY_TYPE | tr "[A-Z]" "[a-z]")/$DYNAWO_BUILD_TYPE_THIRD_PARTY$SUFFIX_CX11
```

Comment:
Could be you please explain the rationale behind this inversion ?

Reply:
Before we could export DYNAWO_THIRD_PARTY_INSTALL_DIR to install in a precise location, but with the current behavior it would be change with some suffix with the type of build. I prefer the old possibiity. For regular user it is completly transparent.